packages/dapp/src/components/Dashboard/ActivityList.tsx

```diff
@@ -2,10 +2,15 @@ import * as React from "react";
 import { Set } from "immutable";
 import BigNumber from "bignumber.js";
 import { ChallengeActivityListItem, ActivityListItem, ResolvedChallengeActivityListItem } from "./ActivityListItem";
+import {
+  ResolvedAppealChallengeActivityListItem,
+  // AppealChallengeActivityListItem,
```

Comment:
Is this import intentionally commented out?

Reply:
yeah just to remind us to take a look at it during refactor